packages/flutter/lib/src/widgets/image.dart

```diff
@@ -29,6 +29,37 @@ export 'package:flutter/painting.dart' show
   MemoryImage,
   NetworkImage;
 
+/// Signature used by [Image] to build a representation of the image's loading
+/// progress.
+///
+/// This is useful for images that are incrementally loaded (e.g. over a local
+/// file system or a network) and you want to give the user an indication of
```

Comment:
Avoid 'you'.

Reply:
I can change it, but pourquoi (I see 152 files in flutter/lib and flutter_test/lib) with "you" in the comment.

Reply:
Either way, done 😄